### ISSUE TITLE
Does not work in a scrollview

### ISSUE THREAD

**Issue by AugustDev** [jaywcjlove/swiftui-markdown#10]
`Markdown` seems to have a scrollview of it's own. It would make sense for a library component to occupy as much space as needed and allow scrolling behaviour to be delegated to application developers.

It seems right now the swiftui-markdown package is useful only in cases where we want Markdown content to occupy the full screen as in the examples?

**Comment by marklambourne** [comment]
Yep, same for me. It's a shame, it severely limits its usefulness. :-(
